### DISCUSSION TITLE
Doppio controllo pwm teensy

### DISCUSSION THREAD

**Discussion by distebia** [phil-barrett/grblHAL-teensy-4.x#99] [Q&A]
Salve, ho visto che con la build del 10/12/23 è stata abilitata la possibilità di avere + mandrini (anche pwm) collegati e gestiti da grblhal. Il teensy e la T41U5XBB sono in grado di gestire 2 uscite pwm indipendenti? se si quali pin potrei usare?

**Comment by phil-barrett** [comment]
No, the T41U5XBB only supports 1.

**Reply by distebia** [reply]
quindi non posso prendere nessuna altra uscita da nessun altro pin?

**Reply by phil-barrett** [reply]
You will have to make changes to grblHAL.  It might be possible to use the direction pin or enable.

**Reply by phil-barrett** [reply]
You might want to ask Terje about how to do that.
